frameworks/D/vibed/benchmark_config.json

```diff
@@ -7,18 +7,20 @@
       "db_url": "/db",
       "query_url": "/queries?queries=",
       "plaintext_url": "/plaintext",
+      "fortune_url": "/fortunes",
+      "update_url": "/updates?queries=",
       "port": 8080,
       "approach": "Realistic",
       "classification": "Platform",
       "database": "MongoDB",
       "framework": "vibed",
       "language": "D",
-      "orm": "Raw",
+      "orm": "Micro",
       "platform": "D",
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "Vibe.D",
+      "display_name": "vibe.d/MongoDB",
```

Comment:
Drop the "/MongoDB" - that is displayed in a different way on the results website.